app/assets/javascripts/app/payment.js

```diff
@@ -42,7 +42,13 @@ App.addChild('Payment', _.extend({
   },
 
   selectNationalPayment: function() {
-    this.onTabClick({currentTarget: this.$('#MoIP')});
+    $tab_target = $('#payment-methods').data('preferred-engine');
+
+    if($tab_target == '#') {
+      $tab_target = $('#MoIP');
+    }
+
+    this.onTabClick({currentTarget: this.$($tab_target)});
```

Comment:
You shouldn't call event handlers like this. That is this supposed to do? Can you use jQuery's trigger here? http://api.jquery.com/trigger/